qa/test-old-engine/pom.xml

```diff
@@ -410,6 +410,8 @@
 
                 <exclude>**/ManagementServiceTableCountTest.java</exclude>
 
+                <!-- Can be removed in Camunda 7.23-->
+                <exclude>**/ConcurrentTelemetryConfigurationTest.java</exclude>
```

Comment:
💭 It's not clear to me why the test is excluded. The [PR description](https://github.com/camunda/camunda-bpm-platform/pull/4544#issue-2476199872) only states that "it's not relevant anymore". For someone discovering the exclude it's hard to learn why this test was excluded. Maybe you can add a bit more background in the PR?

Reply:
The test is about concurrent telemetry configuration and it has been removed along with the feature in 7.22. However, in `old-engine` setup we execute 7.21 tests (where the test still exist) with the new database schema. Therefore, I decided to exclude it. Now I have second thoughts, I will check further before proceeding with the merge.

Reply:
Ok, so the scenario in the test is about inserting the telemetry property in the database. As for new instances that scenario is valid. In rolling upgrades scenario (which `old-engine` scenario validates), the property already exist in the database so this test is no longer relevant.
I will extend the comment for the exclude.